Projects and About Me Sections

Baayeh opened the pull request:
This feature was purposely created to display my projects and add more details about the languages, frameworks, and skills I have.

Baayeh committed:
Added the Projects section to the index page to list all projects done

Baayeh committed:
Added the section that talks about myself - About Me section

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+            <section class="project-list">
+                <section class="pj-card pj-1">
+                    <section class="pj-thumbnail"></section>
+                    <section class="pj-card-body">
```
- [ **OPTIONAL** ] For good semantics I think it would be better if you use the `div` tags on each **project's card** instead of a `section` tag.

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+
+        <!--Projects Section-->
+        <section id="projects">
+            <h3 class="pj-heading">My Recent Works</h3>
+
```
- [ ] Kindly make sure you are using the correct color like the one used on your chosen Figma template design.

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+                <section class="pj-card pj-1">
+                    <section class="pj-thumbnail"></section>
+                    <section class="pj-card-body">
+                        <h2 class="pj-card-title">Multi-Post Stories Gain+Glory</h2>
+                        <ul class="pj-card-lists">
```
- [ ] Kindly make sure you are using the correct **color** on each project title like the one used on your chosen Figma template design.

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+
+        <!--About Me-->
+        <section id="about-me">
+            <section class="about-text">
+                <h3 class="about-heading">About me</h3>
```
- [ ] Kindly make sure you are using the correct color like the one used on your chosen Figma template design.

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+                <h3 class="about-heading">About me</h3>
+                <img id="about-icon" src="assets/Mask_Group.svg" alt="Background design for about section">
+                <p class="about-desc"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <img id="about-bg" src="assets/aboutFrame.svg" alt="">
```
- [ ] As required for the project, kindly make sure all the decorations(the selected images) are always used as `Background images`(i.e. in the style.css file) and not using the `<img>` tags on them.

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+                <img id="about-icon" src="assets/Mask_Group.svg" alt="Background design for about section">
+                <p class="about-desc"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <img id="about-bg" src="assets/aboutFrame.svg" alt="">
+                <button class="about-btn" type="button">Get My Resume</button>
+            </section>
```
- [ ] Kindly make sure you use the correct values of `font size, line height,...` on **buttons text** like the one used on the Figma design to match the given template design.

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+                                <span>CSS</span>
+                            </li>
+                        </ul>
+                    </section>
+                </li>
+
+                <li class="list-item">
+                    <img src="assets/icon-frameworks.svg" alt="Languages Icon">
+
+                    <section class="list-body">
+                        <h3 class="list-title">Frameworks</h3>
+
+                        <ul class="languages">
+                            <li class="language">
+                                <span>Bootstrap</span>
+                            </li>
+                            <li class="language">
+                                <span>Ruby on rails</span>
+                            </li>
+                        </ul>
+                    </section>
+                </li>
+
+                <li class="list-item">
+                    <img src="assets/icon-skills.svg" alt="Languages Icon">
+
+                    <section class="list-body">
+                        <h3 class="list-title">Skills</h3>
+
+                        <ul class="languages">
+                            <li class="language">
+                                <span>GitHub</span>
+                            </li>
+                            <li class="language">
+                                <span>Terminal</span>
+                            </li>
+                        </ul>
+                    </section>
+                </li>
+            </ul>
```
- [ ] Kindly make sure each title such as(**Languages, Skills and Frameworks**) is capitalized like in the Figma design and not in **Uppercase**.

- [ ] Kindy make sure each category has all the items in each card(e.g.: For Frameworks: Ruby on Rails, Bootstrap, Rspec, Selenium,...). Make sure you apply this approach to each card category.

tresorsawasawa commented on index.html:
```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+        </section>
+
+        <!--About Me-->
+        <section id="about-me">
+            <section class="about-text">
+                <h3 class="about-heading">About me</h3>
+                <img id="about-icon" src="assets/Mask_Group.svg" alt="Background design for about section">
+                <p class="about-desc"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <img id="about-bg" src="assets/aboutFrame.svg" alt="">
+                <button class="about-btn" type="button">Get My Resume</button>
+            </section>
+
+            <ul class="categories">
+                <li class="list-item">
+                    <img src="assets/icon-languages.svg" alt="Languages Icon">
+
+                    <section class="list-body">
+                        <h3 class="list-title">Languages</h3>
+
```
- [ ] Kindly make sure you are using the correct `color` on all **the texts** like the one provided on the Figma template

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Baayeh 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Baayeh committed:
Changed the section tags on each project' card to div tags

Baayeh committed:
Removed the images in the about me section and added them as a background

Changed the colors and font sizes of all the headings and texts to suit the appropriate template

Modified all buttons to suit the template provided

mugishasam123 reviewed (approved):
Hi @Baayeh  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors 🥇 
- Nice use of Github workflow 👍 
- Descriptive commit messages, keep rocking 🚀 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
